extra-dev/packages/coq-matching-logic/coq-matching-logic.dev/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+maintainer: "jenda.tusil@gmail.com"
+version: "dev"
+
+homepage: "https://github.com/harp-project/AML-Formalization"
+dev-repo: "git+https://github.com/harp-project/AML-Formalization.git"
+bug-reports: "https://github.com/harp-project/AML-Formalization/issues"
+license: "LGPL"
```

Comment:
Which version of LGPL is this? Please use an identifier from here: https://spdx.org/licenses/